cfme/tests/configure/test_access_control.py

```diff
@@ -2101,11 +2120,43 @@ def test_tenant_visibility_miq_ae_namespaces_all_parents():
     pass
 
 
-@pytest.mark.manual
+@pytest.fixture()
+def create_user(appliance):
+    """This fixture will create user, group, and role with all product feature
+    by clicking individually"""
+    def _create_user():
+        features = FEATURES_511 if appliance.version > "5.11" else FEATURES_510
+        product_features = [[["Everything", feature], True] for feature in features]
+        role = appliance.collections.roles.create(
+            name=fauxfactory.gen_alpha(15, start="API-role-"),
+            product_features=[[["Everything"], False]] + product_features,
+        )
+        # creating group
+        group = appliance.collections.groups.create(
+            description=fauxfactory.gen_alphanumeric(22, "group_description_"),
+            role=role.name,
+        )
+
+        creds = Credential(
+            principal=fauxfactory.gen_alphanumeric(start="user_"),
+            secret=fauxfactory.gen_alphanumeric(),
+        )
+        # user with above group
+        user = appliance.collections.users.create(
+            name=fauxfactory.gen_alphanumeric(start="user_"),
+            credential=creds,
+            email=fauxfactory.gen_email(),
+            groups=group,
+        )
+        return role, group, user
+    return _create_user
+
+
 @test_requirements.rbac
-@pytest.mark.meta(coverage=[1684472, 1693720])
+@pytest.mark.meta(automates=[1684472, 1693720])
```

Comment:
The clone will be resolved automatically, if you want you can just list one of these BZs.